Utils.py

```diff
@@ -436,6 +436,8 @@ def find_class(self, module: str, name: str) -> type:
             obj = getattr(mod, name)
             if issubclass(obj, self.options_module.Option):
                 return obj
+            elif issubclass(obj, self.options_module.PlandoConnection):
+                return obj
```

Comment:
Could do this instead I believe:
```suggestion
            if issubclass(obj, (self.options_module.Option, self.options_module.PlandoConnection)):
                return obj
```

Reply:
Tested this now, it works